MedallionShell/Streams/MergedLinesEnumerable.cs

```diff
@@ -1,15 +1,20 @@
 ﻿using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.IO;
 using System.Linq;
-using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Medallion.Shell.Streams
 {
-    internal sealed class MergedLinesEnumerable : IEnumerable<string>
+    internal sealed class MergedLinesEnumerable :
+#if NETCOREAPP3_0_OR_GREATER || NETSTANDARD2_1_OR_GREATER
```

Comment:
We're adding Microsoft.Bcl.AsyncInterfaces for netstandard2.0 but then here we're not implementing the interface in that framework. Seems off...

Reply:
I was confused by https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.iasyncenumerable-1?view=net-8.0#applies-to. `IAsyncEnumerable<T>` is a built-in API in netcoreapp3.0+ and netstandard2.1, but we're adding `Microsoft.Bcl.AsyncInterfaces` 8.0.0 for netstandard2.0 ([which is available in net462+ and netstandard2.0](https://www.nuget.org/packages/Microsoft.Bcl.AsyncInterfaces/8.0.0#dependencies-body-tab)), so the correct condition would be netcoreapp3.0+ and netstandard2.0.